GeneratorInterface/GenFilters/interface/MCParticlePairFilter.h

```diff
@@ -38,6 +38,9 @@
 namespace edm {
   class HepMCProduct;
 }
+namespace HepMC {
```

Comment:
@davidlange6 sorry for the frequent pings... Could you tell me explicitly how I should change the code?
- removing the "class FourVector;" line is not an option, because it does no longer compile.
- I could instead include "SimDataFormats/GeneratorProducts/interface/HepMCProduct.h" (it is currently in the .cc)
- I could instead include "HepMC/SimpleVector.h"